### PR TITLE
Update behaviors.js

### PR DESCRIPTION
**Pull request by wvankuipers**
Fixed a small typo

### CODE REVIEW

File: src/behaviors.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ Marionette.Behaviors = (function(Marionette, _) {
         behaviorEvents = Marionette.normalizeUIKeys(behaviorEvents, getBehaviorsUI(b));
 
         var j = 0;
-        _.each(behaviorEvents, function(behaviour, key) {
+        _.each(behaviorEvents, function(behaviorEvent, key) {
```
Comment (ahumphreys87) [inline src/behaviors.js]: I'm not sure `behaviorEvent` is the correct term here either. Maybe `handler`?

Comment (wvankuipers) [inline src/behaviors.js]: It's a item from the behaviorEvents collection, so behaviorEvent seems to be the correct phrase. Handler could also be working, but I find it a bit more abstract.

Comment (ahumphreys87) [inline src/behaviors.js]: I think the actual event is the 'key' though. In this case 'behaviorEvent'
is what actually responds to the event.

Comment (wvankuipers) [inline src/behaviors.js]: Noted, maybe `eventHandler` is the right therm then?

Comment (ahumphreys87) [inline src/behaviors.js]: Yeah sounds good to me :)

**Comment by wvankuipers** [comment]
@ahumphreys87 Done!

**Comment by jasonLaster** [comment]
that's fine, Andrew is being nice because he's british. `behaviour` is a perfectly valid spelling in my book :)

**Comment by wvankuipers** [comment]
@jasonLaster Yes it is, but it's the only time it's being used in the entire codebase! I noticed because I couldn't find it using a search ;)

**Comment by blurrah** [comment]
+1 for Marionette's independence day!

**Comment by ahumphreys87** [comment]
Hehe I hadn't even realised it was the british spelling! I'm a convert now - too many PR comments making me change spelling :stuck_out_tongue:

**Comment by paulfalgout** [comment]
Let's get these commits squashed and I'm :+1:

**Comment by wvankuipers** [comment]
Here you go

**Comment by jasonLaster** [comment]
Thanks @wvankuipers. 

@marionettejs/marionette-core note the PR is against master and not `patch`

**Comment by jasonLaster** [comment]
Thanks! it's in

**Pull request closed without merging**